unlock-app/src/__tests__/pages/paywall.test.js

```diff
@@ -78,4 +78,28 @@ describe('Paywall', () => {
       expect(unlockPage).toHaveBeenCalled()
     })
   })
+  describe('the unlocked flag', () => {
+    it('should be present when the paywall is unlocked', () => {
+      const { queryByText } = rtl.render(
```

Comment:
sexy use of destructuring!

Reply:
Perhaps my favorite feature in JS right now! (Clojure has good destructuring too)